### ISSUE TITLE
Support "Connection" property

### ISSUE THREAD

**Issue by Aubron** [dniel/traefik-forward-auth0#321]
Per [Auth0 Docs](https://auth0.com/docs/api/authentication#login), it looks like there's a "Connection" property, that could allow one to throw straight to an expected social provider. It seems like it should be straightforward to map a configuration to provide this if specified.

Would be happy to contribute a PR if this sounds viable.

**Comment by dniel** [comment]
yeah,. should be easy enough to add, per application or as a global setting? If you add ass a PR I'll check it out and release it.
